wp-content/plugins/core/src/Blocks/Types/Gallery_Slider/Gallery_Slider.php

```diff
@@ -0,0 +1,111 @@
+<?php declare(strict_types=1);
+
+namespace Tribe\Project\Blocks\Types\Gallery_Slider;
+
+use Tribe\Libs\ACF\Block;
+use Tribe\Libs\ACF\Block_Config;
+use Tribe\Libs\ACF\Field;
+use Tribe\Libs\ACF\Field_Section;
+
+class Gallery_Slider extends Block_Config {
+
+	public const NAME = 'galleryslider';
+
+	public const SECTION_CONTENT  = 's-content';
+	public const SECTION_SETTINGS = 's-settings';
+
+	public const TITLE       = 'title';
+	public const DESCRIPTION = 'description';
+	public const CTA         = 'cta';
+	public const GALLERY     = 'gallery';
+
+	public const IMAGE_RATIO = 'image_ratio';
+	public const FIXED       = 'fixed';
+	public const VARIABLE    = 'variable';
+
+	public const CAPTION_DISPLAY      = 'caption_display';
+	public const CAPTION_DISPLAY_SHOW = 'caption_display_show';
+	public const CAPTION_DISPLAY_HIDE = 'caption_display_hide';
+
+	public const SLIDESHOW = 'slideshow';
+
+	public function add_block() {
+		$this->set_block( new Block( self::NAME, [
+			'title'       => __( 'Gallery Slider', 'tribe' ),
+			'description' => __( 'A custom block by Modern Tribe', 'tribe' ), // TODO: describe the block
+			'icon'        => '<svg enable-background="new 0 0 146.3 106.3" version="1.1" viewBox="0 0 146.3 106.3" xml:space="preserve" xmlns="http://www.w3.org/2000/svg"><style type="text/css">.st0{fill:#16D690;}.st1{fill:#21A6CB;}.st2{fill:#008F8F;}</style><polygon class="st0" points="145.2 106.3 72.6 42.3 26.5 1.2 0 106.3"/><polygon class="st1" points="145.2 106.3 0 106.3 72.6 42.3 118.6 1.2"/><polygon class="st2" points="72.6 42.3 145.2 106.3 0 106.3"/></svg>', // TODO: set SVG icon
+			'keywords'    => [ __( 'gallery', 'slider', 'carousel', 'image', 'tribe' ) ], // TODO: select appropriate keywords
```

Comment:
Recognizing that this is TODO only, but the current structure will break translations. Each key word should be init's own translation call, or they should all share the same translation string + the `'tribe' text domain as the second argument. 

Example: `__( 'gallery, slider, carousel, image', 'tribe' )`